content/kapp-controller/docs/latest/package-consumption.md

```diff
@@ -160,20 +160,20 @@ spec:
           - stranger
   template:
     spec:
-      deploy:
-      - kapp: {}
       fetch:
       - imgpkgBundle:
-          image: index.docker.io/k8slt/kctrl-example-pkg@sha256:73713d922b5f561c0db2a7ea5f4f6384f7d2d6289886f8400a8aaf5e8fdf134a
+          image: k8slt/kctrl-example-pkg:v1.0.0
```

Comment:
Interesting question here: Do we want to show this in digest format since technically kbld will handle this during templating phase? Is it valuable to call more attention to what kbld is doing to educate first time users?

Reply:
I think leaving it in digest form makes sense. The in cluster resource will likely have the digest, so the docs should be as close to that as possible.

Reply:
Addressed